src/spec/settings_spec.js

```diff
@@ -132,4 +132,29 @@ describe('Settings Page', () => {
     expect(settings.getDefaultDomain()).not.toEqual(defaultDomain);
     expect(settings.getDefaultDomain()).toEqual("fromdoppler.com");
   });
+
+  it('should set as enabled domain correctly', () => {
+    // Arrange
+    beginAuthenticatedSession();
+    browser.addMockModule('descartableModule2', () => angular
+      .module('descartableModule2', ['ngMockE2E'])
+      .run($httpBackend => {
+        $httpBackend.whenGET(/\/accounts\/[\w|-]*\/domains/).respond(200, {
+          "domains": [{name: "relay.com"}, {name: "fromdoppler.com" }, {name: "makingsense.com", disabled: true }, {name: "makingsense12.com" }],
+          "defaultDomain": "relay.com"
+        });
+        $httpBackend.whenPUT(/\/accounts\/[\w|-]*\/domains/).respond(201, {

```

Comment:
Is it possible to also test the request body? It could be useful to understand the test and for validate if the system is doing what we expect